flink/v1.16/flink/src/main/java/org/apache/iceberg/flink/TableLoader.java

```diff
@@ -76,6 +80,17 @@ public Table loadTable() {
       return tables.load(location);
     }
 
+    @Override
+    @SuppressWarnings("checkstyle:NoClone")
+    public TableLoader clone() {
+      try {
+        return (TableLoader) super.clone();
```

Comment:
why not just `return new HadoopTableLoader(location, new Configuration(hadoopConf.get())`?